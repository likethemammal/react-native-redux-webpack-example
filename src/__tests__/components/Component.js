import React from 'react';
import {shallow} from 'enzyme';

import general_components from '@likethemammal/general-components'

const shared = general_components.__tests__._shared.tests

import SomeComponent from '../../components/App.unit';

describe('SomeComponent', () => {

    const component = shallow(
        <SomeComponent />
    )

    shared.SNAPSHOTS_SHOULD_MATCH([
        component
    ])
})