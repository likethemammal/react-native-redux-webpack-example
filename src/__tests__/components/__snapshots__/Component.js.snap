// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent Snapshot 0 should match 1`] = `
<div>
  <div
    onClick={[Function]}
  >
    Click Me!
  </div>
  <View
    style={
      Object {
        "backgroundColor": "lightblue",
        "flex": 1,
        "padding": 0,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "rgba(0,0,0,0.3)",
          "borderWidth": 1,
          "flexDirection": "row",
          "height": 80,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#333",
          }
        }
      >
        Hello
      </Text>
    </View>
  </View>
  <t
    className="SVG-container"
    raw="test-file-stub"
    style={
      Object {
        "display": "none",
      }
    }
  />
</div>
`;
